config-files/config-aggregator.yaml

```diff
@@ -35,6 +35,9 @@ aggregator:
   enable_metrics: true
   metrics_ip_port_address: localhost:9091
   telemetry_ip_port_address: localhost:4001
+  garbage_collector_period: 60s #The period of the GC process #TODO make it 1 day
+  garbage_collector_tasks_age: 3 #The age of tasks that will be removed by the GC #TODO make it 1 month
```

Comment:
Missing TODOs. Will we do them now or leave for admin to do?

Reply:
Left recommended values for devops to set on prod